### DISCUSSION TITLE
openssl vulnerabilities in alpine base image

### DISCUSSION THREAD

**Discussion by Hark0q** [qdm12/gluetun#1454] [Q&A] [answered]
Hi, i'm pretty new to docker and wondering how to address vulnerabilities in the underlying alpine base image used by gluetun.

My understanding is the latest release of gluetun is built from alpine/v3.16 which has some vulns. They are addressed in alpine 3.17.2.

I assume the reason gluetun lags the latest alpine build is due to testing? 

Is there a way I can build a local version of gluetun using the latest alpine release myself? 

Any advice appreciated or links I can read up on.

**Comment by bnhf** [comment]
@Hark0q 

So I just exec'd into a Gluetun container and ran an ```apk update```, and it looks like gluetun:latest is based on Alpine 3.17.2 to me. :-)

```console
/ # apk update
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/aarch64/APKINDEX.tar.gz
v3.17.2-194-gec14c5f7a35 [https://dl-cdn.alpinelinux.org/alpine/v3.17/main]
v3.17.2-195-gc56b42b1969 [https://dl-cdn.alpinelinux.org/alpine/v3.17/community]
OK: 17684 distinct packages available
```

**Comment by qdm12** [comment, marked as answer]
The latest tag `qmcgaw/gluetun` has alpine 3.17 and openssl 3, I need to fix (for vpn unlimited) before doing a release, so you can use that one for now until next release tag.
